src/model/mongodb/mongodb-repository.ts

```diff
@@ -106,7 +106,7 @@ class MongoDBArticleRepository implements ArticleRepository {
   }
 
   async getArticleVersion(identifier: string): Promise<EnhancedArticleWithVersions> {
-    const allVersions = await this.versionedCollection.find({ $or: [{ _id: identifier }, { msid: identifier }] })
+    const allVersions = await this.versionedCollection.find({ msid: identifier })
```

Comment:
If I understand this correctly, this is just clearing up. No need to use the `_id` identifier any more. If there are other things like this in this PR such as the removal of the test in `api.test.ts` then we should get them in a separate PR. There is enough to review here without the noise of unrelated changes.

Reply:
Me and Elliot chatted about this yesterday. I think the conclusion was that perhaps it's not correct to remove this access pattern, but that it's probably not doing it right as was, and will need refining.

Separately, we want to remove returning all the content, and just return metadata, for the less interesting versions, but that's an easy follow up.